### PR TITLE
LPS-88715 Add integration tests for adaptive media

### PR DESCRIPTION
**Pull request by ruben-pulido**

### CODE REVIEW

**Comment by ruben-pulido** [comment]
ci:test:sf <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by ruben-pulido** [comment]
ci:test:relevant <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/headless-apio/media-object/media-object-apio-test/src/testIntegration/java/com/liferay/media/object/apio/internal/architect/resource/test/DefaultMediaObjectNestedCollectionResourceTest.java [hunk 1]
```diff
@@ -91,22 +110,128 @@ public void testAddMediaObject() throws Exception {
...
+
+		Assert.assertEquals(
+			adaptiveMediaList.toString(), 2, adaptiveMediaList.size());
+
+		AdaptiveMedia<AMImageProcessor> adaptiveMedia1000px =
+			adaptiveMediaList.get(0);
+
+		URI contentURL1000px = adaptiveMedia1000px.getURI();
+
+		URL url1000px = new URL(TestPropsValues.PORTAL_URL + contentURL1000px);
```
Comment (cgoncas) [inline modules/apps/headless-apio/media-object/media-object-apio-test/src/testIntegration/java/com/liferay/media/object/apio/internal/architect/resource/test/DefaultMediaObjectNestedCollectionResourceTest.java]: Is contentURL1000px only used here? Replace by adaptiveMedia1000px.getURI()

Comment (ruben-pulido) [reply modules/apps/headless-apio/media-object/media-object-apio-test/src/testIntegration/java/com/liferay/media/object/apio/internal/architect/resource/test/DefaultMediaObjectNestedCollectionResourceTest.java]: 👍

**Pull request closed without merging**